embedded/xsens_imu/example/Makefile

```diff
@@ -0,0 +1,20 @@
+CC            = gcc
+CFLAGS        = -Wall -Wextra
+LIBS          = -lpthread -lrt
+
+all: test
```

Comment:
This Makefile will work but I strongly recommend looking at our other build systems as reference designs for how to make sure your dependency trees are accurate and your project can scale to the complexity you expect in the end (good to plan for arbitrary complexity).

Here are the [compiling rules for my senior design firmware](https://github.com/vkottler/senior-design/blob/master/src/mk/rules.mk) and [here's an example of a top-level Makefile](https://github.com/vkottler/telemetry/blob/master/Makefile) I'm using to build a static library for telemetry.

Reply:
Makes sense for bigger projects absolutely. For this example I was going for a fast + simple solution